### ISSUE TITLE
Exception thrown on generation of tabular data from REFPROP: "PHI0dll function is not available in your version of REFPROP..."

### ISSUE THREAD

**Issue by mapipolo** [CoolProp/CoolProp#830]
Using: nightly build of 32-bit C# DLL (downloaded October 6)
On: Windows 32-bit, C#; REFPROP 9.1

I'm running the following line of code to try and generate the tabular data needed for the interpolation methods.

`AbstractState tabState = AbstractState.factory("TTSE&REFPROP", "R23");`

Upon execution, AbstractState.cs always throws the following exception (but only after making some progress on table generation, as seen in the console):

    A first chance exception of type 'System.ApplicationException' occurred in FluidProps.dll
    Additional information: PHI0dll function is not available in your version of REFPROP. Please upgrade

Is there something I'm overlooking? Is the tabular data generation working properly in the latest build?

**Comment by jowr** [comment]
Well, I would guess the error message is right. Have you tried using REFPROP 9.11? If you have a license for 9.1, you should also be entitled to get 9.11 free of charge. Please contact NIST about that and have a look at their website.

**Comment by ibell** [comment]
And if you need the source code with refprop, and you have a valid license,
email me at ian.bell@nist.gov

**Comment by mapipolo** [comment]
That was my first intuition, jowr, but NIST seems to be advertising REFPROP 9.1 as the latest version (http://www.nist.gov/srd/nist23.cfm). I noticed Ian referred to 9.11 once in another issue but wasn't sure if this was a typo. Does one have to contact NIST directly to get 9.11?

**Comment by ibell** [comment]
Yes, you can send an email to ian.h.bell@gmail.com and I can point you in
the right direction.

**Comment by ibell** [comment]
Ok, go ahead and close this issue.
